tests/test_archetypes.py

```
@@ -0,0 +1,548 @@
+        baseline1 = baseline_from_description("warrior")
+        baseline2 = baseline_from_description("calm warrior")
+        
+        # Calm version should have lower overall activation
+        sum1 = sum(baseline1)
+        sum2 = sum(baseline2)
+        assert sum2 <= sum1
+    
+    def test_empty_description(self):
+        """Empty description should default to guardian."""
+        baseline = baseline_from_description("")
+        guardian_baseline = ARCHETYPES["guardian"]["baseline"]
```

Ruff (0.15.0)

[error] Local variable `guardian_baseline` is assigned to but never used

Remove assignment to unused variable `guardian_baseline`

(F841)